docs/meshcloud.customer.md

```diff
@@ -118,7 +117,7 @@ Before a meshCustomer may be deleted, a check is performed to verify the followi
 - all published Service Brokers in the meshCustomer have been [deactivated](meshstack.meshmarketplace.development.md#deactivation-of-service-brokers)
 - the meshCustomer is not a meshPartner. Deletion of meshPartners is not supported.
```

Comment:
there is only one meshPartner so I think we can deleted this

Reply:
but is there a chance there will be two some day? Based on our code it was planned to have more than partner back then. It just not been used from our customers. I wonder if we will do it in the future. 
Depending on how the product should work I will look out for the doc section which is indicating that there can be more than one partner and adapt it within this pr too

Reply:
At the moment there are no plans to have more than one partner and no one uses at the moment more than one partner. So lets remove it in documentation to avoid confusion.